src/main/java/org/jenkinsci/plugins/ewm/actions/ExwsAllocateActionWrapper.java

```diff
@@ -0,0 +1,51 @@
+package org.jenkinsci.plugins.ewm.actions;
+
+import hudson.model.Actionable;
+import hudson.model.Run;
+import jenkins.model.RunAction2;
+import org.jenkinsci.plugins.ewm.Messages;
+
+/**
+ * A {@link RunAction2} that acts as a wrapper to hold multiple {@link ExwsAllocateActionImpl}s.
+ *
+ * @author Alexandru Somai
+ */
+public class ExwsAllocateActionWrapper extends Actionable implements RunAction2 {
```

Comment:
I knew that having an Actionable Action is a bad idea :) But I don't know what I was thinking when I implemented that.